doc/decisions/auth.md

```diff
@@ -0,0 +1,76 @@
+# Authentication & Authorization
+
+## Problem
+The user wants to log in once and be able to fetch ressources
+from the Nextcloud Instance as well as from PermaplanT database.
+The user needs to authenticate themself (who is the user)
+and get the appropriate authorization (does the user have the right priviliges)
+to perform the action.
+
+**OAuth2.0:** **User** can authorize **client** to fetch resources from **resource server**.
+
+**OpenID Connect:** Extension of OAuth2.0 for authentication
+
+## Constraints
+
+- single account to access PermaplanT and Nextcloud
+
+## Solutions
+
+### Alternative A
+OpenID Connect authentication & authorization flow with:
+
+Authorization server: [Nextcloud OIDC App](https://github.com/H2CK/oidc)
+
+Resource server 1: [Nextcloud OAuth2](https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/oauth2.html)
+
+Resource server 2: PermaplanT backend
+
+Client: PermaplanT frontend
+
+**Notes:**
+> Currently it is not yet possible to use an issued Access Token or ID Token to access resources at the Nextcloud instance it self.
+
+\- [H2CK/oidc](https://github.com/H2CK/oidc)
```

Comment:
I think something went wrong here. Did you want to add the link with the bug? Can we maybe fix this problem?

Reply:
I linked the github repository where this is stated under "Limitations". I can also add https://github.com/H2CK/oidc/issues/87 where this is discussed further.